grass7/raster/r.in.srtm.region/r.in.srtm.region.py

```diff
@@ -448,10 +546,34 @@ def main():
             grass.run_command('r.mapcalc', expression = '%s = round(%s)' % (output, output + '.float'))
             grass.run_command('g.remove', type = 'raster',
                               name = '%s,%s,%s' % (output + '.holes', output + '.interp', output + '.float'),
-                  flags = 'f',
+                              flags = 'f',
                               quiet = True)
 
-    grass.run_command('g.remove', type = 'raster', pattern = pattern, flags = 'f', quiet = True)
+
+    # switch to target location
+    if kv['+proj'] != 'longlat':
+        os.environ['GISRC'] = str(TGTGISRC)
+        # r.proj
+        grass.message(_("Reprojecting <%s>...") % output)
+        if not reproj_res:
+            reproj_res = 30 if srtmv3 else 90
```

Comment:
This assumes without testing that map units are meters. If map units are not meters, convert to meters. Better: fatal error, the user needs to set the resolution in case of reprojection. A suitable target resolution depends not only on the target map units but also on the kind of projection and the current region.

Reply:
I changed it the user have to set the resolution if not longlat

Reply:
Looks good to me now.